wsi/drm/drmdisplay.cpp

```diff
@@ -165,9 +165,9 @@ bool DrmDisplay::GetDisplayAttribute(uint32_t config /*config*/,
                                      HWCDisplayAttribute attribute,
                                      int32_t *value) {
   SPIN_LOCK(display_lock_);
-  if (modes_.empty()) {
+  if (modes_.empty() || !IsConnected()) {
     SPIN_UNLOCK(display_lock_);
-    return PhysicalDisplay::GetDisplayAttribute(config, attribute, value);
+    return false;
```

Comment:
This will break Android where it expects minimum values. We need to look this in more detail.